### ISSUE TITLE
Only half of EQ is enabled

### ISSUE THREAD

**Issue by dllz** [wwmm/pulseeffects#640]
Hey, I have weird issue where only have of the EQ is enabled. 

I have installed LSP and Gstreamer. 

![image](https://user-images.githubusercontent.com/5921586/76962911-5e4a0280-6920-11ea-8db0-605bdfd822f2.png)

Any ideas

**Comment by wwmm** [comment]
First time I see that. It could be some kind of issue in the GTK install or a corrupted configuration. Try to reset the equalizer
![Screenshot from 2020-03-18 10-41-22](https://user-images.githubusercontent.com/16225172/76966632-0b168680-6905-11ea-885a-1d8df618ab32.png)
IF that is not enough try to reset PulseEffects
![Screenshot from 2020-03-18 10-41-29](https://user-images.githubusercontent.com/16225172/76966666-15d11b80-6905-11ea-8733-b77970095675.png)

But it is really strange. It would be good to know which PE version you are using as well as on which distribution and how PE was installed.

**Comment by dllz** [comment]
Thanks so much, the reset of just the EQ worked. I tried recalculating and restarting but never a reset. 

I am using version 4.6.6 running on Ubuntu 19.10 and it was installed I am assuming via apt "sudo apt install pulseeffects" If you are not on apt then I did something else but I do have that command in my bash history.

I then tried to EQ without the other plugins, then got the plugins installed and then went from 30 bands to 10 and then to 12 and then to 15 or something like that, there were a number of band changes and once I was satisfied with the number of bands I noticed I no longer had control over half of them.
